ApollosProject/apollos-shovel
# Pick up updated items by their attributes when shoveling

Steps to test! 

1. Run shovel locally pointed at the core rock instances
2. In the core rock, find and update a content item's attribute.
    - Something like `Summary` or `Image` or a feature. 
3. Rerun the most recent `core_content_item_dag` by clearing it. Not the backfill. 
4. You should see the updates show up in your database. 

Caveats: 
This will only work as long as less than 16 item's attributes are being updated at any given point in time. (https://www.rockrms.com/ask/developing/2169)
